shared-bindings/displayio/Bitmap.c

```diff
@@ -45,11 +45,11 @@
 //| the bitmap data is packed into the memoryview with unspecified padding.
 //|
 //| A Bitmap can be treated as a buffer, allowing its content to be
-//| viewed and modified using e.g., with ``ulab.numpy.frombuffer``,
+//| viewed and modified using e.g., with `ulab.numpy.frombuffer`,
```

Comment:
single backquotes in sphinx causes a cross-reference to be generated. Hopefully that will work in this case.